### PR TITLE
enable fleet cluster deployment

### PR DESCRIPTION
**Pull request by audrastump**
Added functionality to deploy to fleet hub cluster

### CODE REVIEW

File: src/actions/deploy.ts [hunk 1]
```diff
@@ -39,13 +39,14 @@ export async function deploy(
 
    // check manifest stability
    core.startGroup('Checking manifest stability')
+   const resourceType = core.getInput('resource-type') || ''
```
Comment (jim-minter) [inline src/actions/deploy.ts:42]: default to Microsoft.ContainerService/managedClusters ?

**Comment by audrastump** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: src/utilities/manifestStabilityUtils.ts [hunk 2]
```diff
@@ -9,8 +9,16 @@ const POD = 'pod'
 
 export async function checkManifestStability(
    kubectl: Kubectl,
-   resources: Resource[]
+   resources: Resource[],
+   resourceType: string
 ): Promise<void> {
+   // Skip if resource type is microsoft.containerservice/fleets
+   if (resourceType === 'microsoft.containerservice/fleets') {
```
Comment (bfoley13) [inline src/utilities/manifestStabilityUtils.ts:16]: What about case sensitivity here?

**Comment by bfoley13** [comment]
Can you also add to the description of this PR describing the changes your making

**Review by bfoley13 — APPROVED** [review]

**Review by davidgamero — APPROVED** [review]
lgtm. in the future we can add a type or enum for fleet/managedcluster

**Pull request merged**